paint/PAINT_Program.cpp

```diff
@@ -105,7 +108,10 @@ void Program::run() const
 
 			if (e.type == SDL_MOUSEBUTTONDOWN) {
 
+
+				
 				clicked = true;
+				drawWindow->setStartCoord({ xMouse, yMouse });
```

Comment:
Re line 149: these tests are not needed. Just set xPrev and yPrev.
**SHOULD**

Reply:
done